images/win/scripts/Installers/Install-Go.ps1

```diff
@@ -29,36 +29,37 @@ function Install-GoVersion
 
     # Extract the zip archive.  It contains a single directory named "go".
     Write-Host "Extracting Go $latestVersion..."
-    Expand-Archive -Path $goArchPath -DestinationPath "C:\" -Force
+    $toolDirectory = Join-Path $env:AGENT_TOOLSDIRECTORY "go\$latestVersion"
```

Comment:
similar comment as in #848 - should we setup a link from the old location to the new while we communicate with users?

Reply:
Done!